requirements_common.txt

```diff
@@ -1,3 +1,4 @@
+Cython==0.29.20
```

Comment:
I believe this dependency is only used by the solrbuilder (and installed inside its Docker file specifically, here: https://github.com/internetarchive/openlibrary/blob/master/scripts/solr_builder/Dockerfile.olpython ). To the best of my knowledge, the setup.py file is never actually run :/